### PR TITLE
DOCSP-30482: remove wildcard imports

### PR DESCRIPTION
**Pull request by rustagir**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-java/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-30482
**[staging](https://docs-mongodbcom-staging.corp.mongodb.com/kotlin/docsworker-xlarge/DOCSP-30482-remove-wildcards/)**

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [ ] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [ ] Are all the links working?

### CODE REVIEW

File: examples/src/test/kotlin/BuildersTest.kt [hunk 1]
```diff
@@ -10,11 +9,11 @@ import org.bson.BsonObjectId
 import org.bson.Document
 import org.bson.codecs.pojo.annotations.BsonId
 import org.junit.jupiter.api.AfterAll
-import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Assertions.assertEquals
```
Comment (mongoKart) [inline examples/src/test/kotlin/BuildersTest.kt:13]: Q: Are both of these needed?

File: examples/src/test/kotlin/BuildersTest.kt [hunk 2]
```diff
@@ -75,8 +74,8 @@ internal class BuildersTest {
         // :snippet-start: builders
         data class Results(val email: String)
 
-        val filter = and(eq(User::gender.name, "female"), gt(User::age.name, 29))
-        val projection = fields(excludeId(), include("email"))
+        val filter = Filters.and(Filters.eq(User::gender.name, "female"), Filters.gt(User::age.name, 29))
+        val projection = Projections.fields(Projections.excludeId(), Projections.include("email"))
```
Comment (mongoKart) [inline examples/src/test/kotlin/BuildersTest.kt:78]: Q: Do you know whether we have a standard around including Filters/Projects/etc. in these operations for readability? If not, does the compiler require these to be here?

**Review by mongoKart — CHANGES_REQUESTED** [review]
just 2 Q's

File: examples/src/test/kotlin/BuildersTest.kt (hunk 1 above)
Comment (rustagir) [reply examples/src/test/kotlin/BuildersTest.kt:13]: no, only the method imports are needed. will clean up

File: examples/src/test/kotlin/BuildersTest.kt (hunk 2 above)
Comment (rustagir) [reply examples/src/test/kotlin/BuildersTest.kt:78]: I had a conversation with Cory about this, she said we wanted to keep the class names when calling the methods. I'm not sure the reasoning, but that **is** the convention for this documentation

**Review by mongoKart — APPROVED** [review]
Import statements LGTM

**Pull request merged**